src/axom/primal/geometry/BezierCurve.hpp

```diff
@@ -158,14 +228,45 @@ class BezierCurve
   /// Returns the order of the Bezier Curve
   int getOrder() const { return static_cast<int>(m_controlPoints.size()) - 1; }
 
-  /// Clears the list of control points
+  /// Make trivially rational. If already rational, do nothing
+  void makeRational()
```

Comment:
`makeRational`, `makeNonrational`, `isRational`, `getOrder`, `isValid`, `clear`, `getWeight`, and `operator[]` are part of the public interface and I don't see tests for them (perhaps I overlooked them).  Please add tests for these methods.

Reply:
Thank you for the suggestions, I've added a test that verifies `makeRational`, `makeNonrational`, `isRational`, and `getWeight`. The other methods you've mentioned are unchanged between rational and nonrational Bezier curves, and already have tests in `primal_bezier_curve.cpp`.

As for `isValidRational`, I think the best course would be to remove it from the public interface altogether, and make the method private. As of now, it is only run within `SLIC_ASSERT` statements, as I don't think users should be allowed to form invalid rational Bezier curves in the first place, as this would cause a whole slew of issues with the evaluation functions.